lib/puppet/provider/templates/invoke_dsc_resource.ps1.erb

```diff
@@ -51,7 +51,7 @@ $invokeParams = @{
 <% end -%>
   }<% if resource.respond_to?(:dscmeta_module_version) %>
   ModuleName = @{
-    ModuleName      = <%= "\"#{resource.dscmeta_module_name}\"" %>
+    ModuleName      = <%= "\"#{vendored_modules_path}/#{resource.dscmeta_module_name}/#{resource.dscmeta_module_name}.psd1\"" %>
```

Comment:
Right, but looking at this it's not explicit about what the branching is done for. We know that it is for `dscmeta_module_version` but that means what when you are looking at this out of context? :)